AmazonChimeSDKDemo/Podfile

```diff
@@ -4,11 +4,11 @@ platform :ios, '10.0'
 target 'AmazonChimeSDKDemoPods' do
```

Comment:
You will need to rebase with latest development branch, this change should have not be included in your PR.